src/database/adminData.js

```diff
@@ -0,0 +1,12 @@
+const admin = {
+    firstName: 'admin',
+    lastName: 'admin',
+    username: 'admin',
+    email: 'admin@gmail.com',
```

Comment:
I think if you said `email:'admin@barefootnomad.com'` would sound better, unless that email exists on gmail